JsonSwiftson/JsonSwiftson.swift

```diff
@@ -160,21 +160,15 @@ public final class JsonSwiftson {
   Parse JSON text into AnyObject. This function is used internally during initialization.
   Null JSON values are parsed as NSNull objects and not as nil values.
 
-  :returns: An object that can be a Dictionary, Arrays, String, numeric type, boolean or NSNull. Returns nil if parsing failed.
+  - returns: An object that can be a Dictionary, Arrays, String, numeric type, boolean or NSNull. Returns nil if parsing failed.
   
   */
   static func parseRaw(json: String) -> AnyObject? {
-
     if let encoded = json.dataUsingEncoding(NSUTF8StringEncoding) {
-      var error: NSError?
-
-      let parsedObject: AnyObject? = NSJSONSerialization.JSONObjectWithData(encoded,
-        options: NSJSONReadingOptions.AllowFragments,
-        error: &error)
-
-      if error != nil { return nil } // failed to parse data to JSON
-
-      return parsedObject
+      do {
+        return try NSJSONSerialization.JSONObjectWithData(encoded,
+          options: NSJSONReadingOptions.AllowFragments)
+      } catch _ {}
```

Comment:
perhaps here we want to throw and have different errors, but this will break compatibility with existing code

aka: `static func parseRaw(json: String) -> AnyObject?` -> `static func parseRaw(json: String) throws -> AnyObject`

i really like what Argo do with their decoded type for example, see https://github.com/thoughtbot/Argo/blob/master/Argo/Types/Decoded.swift